### PR TITLE
Removed deprecated dependencies is one which can do it all

### PR DESCRIPTION
**Pull request by Borewit**
I noticed you picked some old dependencies.

I think [music-metadata](https://www.npmjs.com/package/music-metadata) will handle all file types.

If it doesn't work for you, let me know!

### CODE REVIEW

**Comment by gram7gram** [comment]
Thank you, @Borewit , for the effort! I am glad to know there is such a tool!

I liked how you have went extra step to make me know about it

**Comment by Borewit** [comment]
> I am glad to know there is such a tool!

Is there anything I can do, to help users like yourself to find [music-metadata](https://www.npmjs.com/package/music-metadata)?

**Comment by Borewit** [comment]
I made a mistake, `metadata.common.duration` should be `metadata.format.duration`

File: audio.js [hunk 1]
```diff
@@ -33,25 +32,9 @@ for (let i = 0; i < sources.length; i++) {
...
-
-			musicData(dir + '/' + file, (e, duration) => {
-
-                if (e) throw e
-
-                onComplete(file, Number((duration * 1000).toFixed(4)))
-			})
-		}
+		musicData.parseFile(path.join(dir, file)).then(metadata => {
+			const duration = Number((metadata.common.duration * 1000).toFixed(4))
```
Comment (Borewit) [inline audio.js]: Should be: `metadata.common.duration`

**Pull request merged**